OpenRA.Game/Network/OrderIO.cs

```diff
@@ -31,13 +31,14 @@ public static List<Order> ToOrderList(this byte[] bytes, World world)
 			return ret;
 		}
 
-		public static byte[] SerializeSync(int sync)
+		public static byte[] SerializeSync(World.SyncHash sync)
 		{
 			var ms = new MemoryStream(1 + 4);
 			using (var writer = new BinaryWriter(ms))
 			{
 				writer.Write((byte)OrderType.SyncHash);
-				writer.Write(sync);
+				writer.Write(sync.Hash);
+				writer.Write(sync.PlayerWinstates);
```

Comment:
`OrderManager.CheckSync` will surely also need changes to account for the new player winstates data...?

Reply:
It looks like it's working fine: out-of-sync exception are reported back to players, server is kept alive... What do you see failing exactly?